builds/azure-pipelines/build-release-java.yml

```diff
@@ -3,7 +3,7 @@ steps:
     displayName: Build library
     inputs:
       mavenPomFile: java-library/pom.xml
-      options: --batch-mode --define=gpg.skip --define=org.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn --update-snapshots
+      options: --batch-mode --define=org.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn --update-snapshots
```

Comment:
Would suggest adding a comment here explaining what the options specified are doing (in the other places too). It's a bit verbose but will make sure there isn't any confusion later on about why they're being used.